src/Sulu/Bundle/AdminBundle/Admin/View/FormOverlayListViewBuilder.php

```diff
@@ -53,6 +54,8 @@ public function setFormKey(string $formKey): FormOverlayListViewBuilderInterface
 
     public function setRequestParameters(array $requestParameters): FormOverlayListViewBuilderInterface
```

Comment:
I think I would also add the `@deprecated` annotation, because then it will also be shown in IDEs that this function should not be used.